lib/driver/pg.js

```diff
@@ -290,8 +290,21 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).nodeify(setType.bind(this));
```

Comment:
Aye, I see. Yes you're totally right this is not a good example to follow, I even don't remember why this looks like it looks now anymore. I will definitely take time on this, I wanted to rework the pg driver and some of the others anyway, this weird functions in functions have been a thorn in the side for a while now. Unfortunately I ran out of time the last month, I'm just saddling again right now.
Sorry for that and thanks for pointing this out :)